admin/src/app/Components/hotels-filter/hotels-filter.component.css

```diff
@@ -0,0 +1,7 @@
+aside {
+    width: 20%;
+    float: left;
```

Comment:
`float` is a kind of old-fashion alignment property, can you try to use something different?